R/misc.R

```diff
@@ -472,3 +472,13 @@ check_case_weights <- function(x, spec) {
   }
   invisible(NULL)
 }
+
+# -----------------------------------------------------------------------------
+check_for_newdata <- function(..., call = rlang::caller_env()) {
+  if (any(names(rlang::enquos(...)) == "newdata")) {
```

Comment:
```suggestion
  if (any(names(list(...)) == "newdata")) {
```
I think we don't need to use quosures here, all we want are the names of things passed in the dots, not the things themselves.